packages/ir-sdk/fern/apis/ir-types-latest/definition/ir.yml

```diff
@@ -77,11 +77,24 @@ types:
       hasPaginatedEndpoints: boolean
       hasFileDownloadEndpoints: boolean
       platformHeaders: PlatformHeaders
+      version: optional<VersionDeclaration>
```

Comment:
The `SDKConfig` is typically for SDK specific things, but I think API Version is something that may be important for non SDK generators too -- thoughts on putting it on the top level object instead? 

Something like: 
```
IntermediateRepresentation: 
  apiVersion: optional<APIVersionScheme>
```

Reply:
This was also what I started with, but figured I'd tuck it inside the SDK config because platform headers seemed somewhat related. Let's go with your suggestion.